### DISCUSSION TITLE
"please report this as a bug"

### DISCUSSION THREAD

**Discussion by osbyrne** [vercel/next.js#62608] [Help]
### Link to the code that reproduces this issue

https://github.com/vercel/next.js/tree/canary

### To Reproduce

`npx create-next-app@latest`

get message: 
```
'\\wsl.localhost\Ubuntu\home\oz'
CMD.EXE was started with the above path as the current directory.
UNC paths are not supported.  Defaulting to Windows directory.
```

- no to TypeScript, ESLint, Tailwind CSS, src/directory, 
- yes to App Router
- no to custom import alias

error message:
```
Aborting installation.
Unexpected error. Please report it as a bug:
 [Error: EPERM: operation not permitted, mkdir 'C:\Windows\next-f1'] {
  errno: -4048,
  code: 'EPERM',
  syscall: 'mkdir',
  path: 'C:\\Windows\\next-f1'
}
```

### Current vs. Expected behavior

it should build the app, no error

### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

```bash
Operating System:
  Platform: win32
  Arch: x64
  Version: Windows 11 Home
Binaries:
  Node: 20.9.0
  npm: N/A
  Yarn: N/A
  pnpm: N/A
Relevant Packages:
  next: 14.0.3-canary.11
  eslint-config-next: N/A
  react: 18.2.0
  react-dom: 18.2.0
  typescript: N/A
Next.js Config:
  output: N/A
```

### Which area(s) are affected? (Select all that apply)

CLI (create-next-app)

**Comment by MonstraG** [comment]
It seems that your WSL just put the project into `C:\Windows\` that whole folder is read/write protected against everyone, and is not a good place to store your project.

The message about `cmd.exe` doesn't come from `npx` but from the `cmd.exe` (I think). Fix that issue first, or manually navigate to any other folder and try creating running `npx` command there.

**Comment by timneutkens** [comment]
Yeah what @MonstraG said seems correct to me 👍

**Comment by JesseKoldewijn** [comment]
This is basically what happens with any command when you try to run CMD inside of your wsl context instead of using your wsl shell. So if you'd use bash for instance to call npx the error you mentioned shouldn't appear.

**Comment by JesseKoldewijn** [comment]
> It seems that your WSL just put the project into `C:\Windows\` that whole folder is read/write protected against everyone, and is not a good place to store your project.
> 
> The message about `cmd.exe` doesn't come from `npx` but from the `cmd.exe` (I think). Fix that issue first, or manually navigate to any other folder and try creating running `npx` command there.

This is also true, but the message it shows I believe actually come from the wsl backend but not too sure since I don't run into them myself too often.
